Djikstra/Djikstra/Headers.h

```diff
@@ -0,0 +1,40 @@
+#pragma once
+
+#include <vector>
+#include <string>
+#include <iostream>
+#include <cstdlib>
+#include <fstream>
+#include <map>
+
+
+namespace Graph
+{
+	struct Edge
+	{
+		int source_id;
+		int target_id;
+		float distance;
+	};
+
+	struct Vertex
```

Comment:
Provide a constructor that accepts a `name`. [Delete copying constructors](https://stackoverflow.com/a/22495199) (use C++11 version of linked solution) to avoid surprises.